ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -17,9 +17,11 @@ option csharp_namespace = "Com.Daml.Ledger.Api.V1.Admin";
 // served by a participant node.
 //
 // The authorization rules for its RPCs are specified on the ``<RpcName>Request``
-// messages as boolean expressions over these two facts:
+// messages as boolean expressions over these facts:
 // (1) ``HasRight(r)`` denoting whether the authenticated user has right ``r`` and
-// (2) ``IsAuthenticatedUser(uid)`` denoting whether ``uid`` is the empty string or equal to the id of the authenticated user.
+// (2) ``IsAuthenticatedUser(uid)`` denoting whether ``uid`` is equal to the id of the authenticated user.
```

Comment:
What if `uid` is the empty string?

Reply:
I will create a follow-up PR to roll it back- it should be there.